src/app/owner/page.tsx

```diff
@@ -1,74 +1,88 @@
 import Image, { StaticImageData } from 'next/image';
 import about from "@images/about/about1.png";
-import AboutHeading from './AboutHeading'; 
-import SubHeading from './SubHeading';
-import Section from './Section';
+import AboutHeading from '../components/OwnerPage/AboutHeading';
+import SubHeading from '../components/OwnerPage/SubHeading';
+import Section from '../components/OwnerPage/Section';
 import workMethodsContent from '../utils/workMethodsContent';
 import { educationAndExperience, certificatesAndAchievements } from '../utils/aboutContent';
+import Container from '../components/Share/Container';
+import Button from '../components/Share/Button';
 
 const About = () => {
   return (
     <div className="min-h-screen bg-white">
-      <main className="max-w-4xl mx-auto p-4">
-        <nav className="text-[16px] mb-4 mt-[70px]">
-          <a href="/" className="font-semibold leading-6 underline" aria-label="Wróć do strony głównej">powrót</a>
-        </nav>
+      <Container>
+        <main>
+          <nav className="text-[16px] mb-4 mt-[70px]">
+            <a href="/" className="font-semibold leading-6 underline hidden md:block" aria-label="Wróć do strony głównej">powrót</a>
+          </nav>
 
-        <p className="font-bold text-sm leading-6 mt-[29px]">O nas // Katarzyna Bober</p>
-        <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px]">Katarzyna Bober</h1>
+          <p className="font-bold text-[16px] text-sm leading-6 mt-[80px] md:mt-[29px]">O nas // Katarzyna Bober</p>
+          <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px]">Katarzyna Bober</h1>
 
-        <div className="flex justify-center mb-[20px]">
-          <Image
-            src={about as StaticImageData}
-            alt="Katarzyna Bober with dog"
-            width={1217}
-            height={600}
-            className="rounded-lg mt-[100px] max-w-full"
-            priority
-          />
-        </div>
+          <div className="flex justify-center mb-[20px]">
+            <Image
+              src={about as StaticImageData}
+              alt="Katarzyna Bober with dog"
+              width={1217}
```

Comment:
![image](https://github.com/user-attachments/assets/6f59f101-cae2-402a-b2f7-dee9b8587520)
To remove the warning from the console, try setting the image size with css:
`<Image
              src={about as StaticImageData}
              alt="Katarzyna Bober with dog"
              className="mt-[100px] h-auto w-[338px] max-w-full rounded-lg tablet:w-[602px] desktop:w-[1217px]"
              priority
            />
`

Reply:
There is still a problem in the console, in addition, I noticed that at the highest resolution the image is blurred. :(